src/discrete-layer/components/map-container/map-container.tsx

```diff
@@ -1,11 +1,39 @@
 import React, { useState } from 'react';
-import { Polygon } from 'geojson';
-import { DrawType } from '@map-colonies/react-components';
+import { Feature, FeatureCollection, Point, Polygon } from 'geojson';
+import { find } from 'lodash';
+import * as turf from '@turf/helpers';
```

Comment:
why import *?
it is bad for tree shaking

Reply:
Done